apps/admin-portal/src/store/actions/authenticate.ts

```diff
@@ -253,6 +254,21 @@ export const handleSignOut = () => (dispatch) => {
     }
 };
 
+/**
+ * Get SCIM2 schemas
+ */
+export const getScimSchemas = (profileInfo: BasicProfileInterface = null) => (dispatch) => {
+    dispatch(setProfileSchemaLoader(true));
+    getProfileSchemas()
+        .then((response: ProfileSchema[]) => {
+            dispatch(setProfileSchemaLoader(false));
+            dispatch(setScimSchemas(response));
+        })
+        .catch((error) => {
```

Comment:
Should remove `error` variable since it's not used in the implementation. Handle it when the `TODO` is handled.

Reply:
Resolved with 1a966a0